assets/wizards/payment/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Payment
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Component, render, Fragment, createElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Material UI dependencies.
+ */
+import HeaderIcon from '@material-ui/icons/Dns';
+
+/**
+ * Internal dependencies.
+ */
+import { withWizard } from '../../components/src';
+import Router from '../../components/src/proxied-imports/router';
+import { PaymentMethod } from './views';
+
+const { HashRouter, Redirect, Route, Switch } = Router;
+
+class PaymentWizard extends Component {
+	state = {
+		customer: {},
+	};
+	componentDidMount = props => {
+		this.retrieve();
+	};
+	retrieve = () => {
+		const { setError, wizardApiFetch } = this.props;
+		return wizardApiFetch( { path: '/newspack/v1/wizard/newspack-payment-wizard/' } )
+			.then( ( { customer, subscription } ) => this.setState( { customer, subscription } ) )
+			.catch( error => setError( error ) );
+	};
+	checkout = () => {
+		const { setError, wizardApiFetch } = this.props;
+		return wizardApiFetch( { path: '/newspack/v1/wizard/newspack-payment-wizard/checkout' } )
+			.then( data => {
+				const stripe = Stripe( data.stripe_publishable_key );
+				stripe
+					.redirectToCheckout( {
+						// Make the id field from the Checkout Session creation API response
+						// available to this file, so you can provide it as parameter here
+						// instead of the {{CHECKOUT_SESSION_ID}} placeholder.
```

Comment:
nitpick: not sure if this comment is needed here

Reply:
Removed in https://github.com/Automattic/newspack-plugin/pull/442/commits/204ec178c4b9962a82d173a3137d676802d0a4ae